### PR TITLE
refactor(python): Added formatter support for the Python module (*one module only*)

### LINKED ISSUE #1208: Show only VIRTUAL_ENV value without version in python module
#### Is your feature request related to a problem? Please describe.
I usually know the python version in the project I'm working on, so I'd like to see only `$VIRTUAL_ENV` value and eventually symbol.

#### Describe the solution you'd like
Version and `$VIRTUAL_ENV` value visibility should be separable. Each of them should have its own configuration.

### PR DESCRIPTION
**Pull request by jimleroyer**
#### Description

This PR refactors the Python module text output by using the new string formatter feature.

#### Motivation and Context

Related to issue #1057
Fixes #1208 

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Screenshots (if appropriate):

image

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [x] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: docs/config/README.md [hunk 1]
```diff
@@ -1208,14 +1208,23 @@ The module will be shown if any of the following conditions are met:
...
+| Option               | Default                                                    | Description                                                                 |
+| -------------------- | ---------------------------------------------------------- | --------------------------------------------------------------------------- |
+| `format`             | `"via [${symbol}${version}( \\($virtualenv\\))]($style) "` | The format for the module.                                                  |
+| `symbol`             | `"🐍 "`                                                     | A format string representing the symbol of Python                           |
+| `style`              | `"yellow bold"`                                            | The style for the module.                                                   |
+| `pyenv_version_name` | `false`                                                    | Use pyenv to get Python version                                             |
+| `pyenv_prefix`       | `"pyenv "`                                                 | Prefix before pyenv version display (default display is `pyenv MY_VERSION`) |
```
Comment (heyrict) [inline docs/config/README.md:1217]: Quickly skimmed through your code, and I didn't find this option used anywhere in your code.

File: docs/config/README.md [hunk 2]
```diff
@@ -1208,14 +1208,23 @@ The module will be shown if any of the following conditions are met:
...
+| `scan_for_pyfiles`   | `true`                                                     | If false, Python files in the current directory will not show this module.  |
+| `disabled`           | `false`                                                    | Disables the `python` module.                                               |
+
+### Variables
+
+| Variable | Example         | Description                          |
+| -------- | --------------- | ------------------------------------ |
+| version  | `"v3.8.1"`      | The version of `python`              |
+| symbol   | `"🐍 "`         | Mirrors the value of option `symbol` |
+| style    | `"yellow bold"` | Mirrors the value of option `style`  |
```
Comment (heyrict) [inline docs/config/README.md:1227]: There are more variables than described here, including `$virtualenv`.

File: src/modules/python.rs [hunk 3]
```diff
@@ -41,25 +42,43 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
...
-    module.set_style(config.style);
-    module.create_segment("symbol", &config.symbol);
-
-    if config.pyenv_version_name {
-        let python_version = utils::exec_cmd("pyenv", &["version-name"])?.stdout;
-        module.create_segment("pyenv_prefix", &config.pyenv_prefix);
-        module.create_segment("version", &SegmentConfig::new(&python_version.trim()));
+    let python_version = if config.pyenv_version_name {
+        let version = utils::exec_cmd("pyenv", &["version-name"])?.stdout;
+        version
```
Comment (heyrict) [inline src/modules/python.rs:47]:
```suggestion
        utils::exec_cmd("pyenv", &["version-name"])?.stdout
```
Suggested by clippy

File: docs/config/README.md (hunk 2 above)
Comment (jimleroyer) [reply docs/config/README.md:1227]: Added `virtualenv` as a variable...

File: docs/config/README.md (hunk 1 above)
Comment (jimleroyer) [reply docs/config/README.md:1217]: True, removed it from this section..

**Comment by andytom** [comment]
@jimleroyer Thank you for your contribution.

**Pull request merged**